app/views/cleaners/show.html.erb

```diff
@@ -1,19 +1,55 @@
-<p id="notice"><%= notice %></p>
+<div class="container">
+  <ol class="breadcrumb">
```

Comment:
OK notice is moved to partial, and to make partial for breadcrumbs need some effort. so I will do it in my free time.